specification/callbacks/channel-subscription-openapi-spec.json

```diff
@@ -14,7 +14,7 @@
     "/nhs-notify-callbacks/v1/channel-status": {
```

Comment:
Do we need to update the doc for message status callbacks as well?

Reply:
Its actually not in this one as all, the description is alot less descriptive in general.